### ISSUE TITLE
Завершение сессии на всех устройства после использования скрипта 

### ISSUE THREAD

**Issue by Anton-Shvetc** [Grigorich-JSDio/TG-Parser#6]
Возникала ли проблемы выхода из тг аккаунта со всех устройства после использования скрита? Есть ли решение данной проблемы?

**Comment by rbalanev** [comment]
> Возникала ли проблемы выхода из тг аккаунта со всех устройства после использования скрита? Есть ли решение данной проблемы?

Да, у меня такая же проблема + ограниченное число участников группы\канала в выгрузке. Не более 50 человек((

**Comment by Xsenix** [comment]
Need to run the client with the following parameters - client = Telegram Client('my_session', api_id, api_hash, system_version="4.16.30-vxCUSTOM")
**Необходимо запустить клиент со следующими параметрами - client = Telegram Client('my_session', api_id, api_hash, system_version="4.16.30-vxCUSTOM")**

The fact is that authentication algorithms do not allow scripts to work using the current Telegram security patch
**Дело в том что алгоритмы аутентификации не позволяет работать скриптам используя текущий патч безопасности Телеграм**

In your case, the code will look something like this:
**В вашем случае код будет выглядеть примерно так:**

my_functions.py
```
import re
import time
import string
import config
from telethon.sync import TelegramClient
from telethon.tl.types import ChannelParticipantsAdmins, ChannelParticipantsSearch
from telethon import functions, errors
api_id = config.api_id
api_hash = config.api_hash
session = 'session.session'
client = TelegramClient('session.session', api_id, api_hash, system_version="4.16.30-vxCUSTOM")
client.start()
```

**Comment by Xsenix** [comment]
check this
https://github.com/LonamiWebs/Telethon/issues/4051#issuecomment-1491747149

**Comment by Xsenix** [comment]
#https://github.com/LonamiWebs/Telethon/issues/4051

**Comment by MirdexData** [comment]
проблема решилась добавлением этих строк?
